scim-core/src/main/java/org/apache/directory/scim/core/repository/PatchGenerator.java

```diff
@@ -205,6 +205,7 @@ private List<PatchOperation> createPatchOperations() throws IllegalArgumentExcep
       JsonNode node2 = objectMapper.valueToTree(resource);
       nullEmptyLists(node2);
       JsonNode differences = JsonDiff.asJson(node1, node2, DiffFlags.dontNormalizeOpIntoMoveAndCopy());
+//      JsonNode differences = JsonDiff.asJson(node1, node2, EnumSet.of(DiffFlags.ADD_EXPLICIT_REMOVE_ADD_ON_REPLACE, DiffFlags.OMIT_MOVE_OPERATION, DiffFlags.OMIT_COPY_OPERATION, DiffFlags.ADD_ORIGINAL_VALUE_ON_REPLACE));
```

Comment:
I think we may need to change the flags used to generate the diffs.
If we do that, i assume some of the logic that maps the diff to a SCIM patch request may need to change